software/firmware/europi.py

```diff
@@ -298,16 +298,17 @@ def _bounce_wrapper(self, pin):
                 return
             self.last_rising_ms = time.ticks_ms()
             return self._rising_handler()
-        elif self.value() == LOW:
+        else:
             if time.ticks_diff(time.ticks_ms(), self.last_falling_ms) < self.debounce_delay:
                 return
             self.last_falling_ms = time.ticks_ms()
 
-            # Check if 'other' pin is set and if 'other' pins is high and if this pin has been high for long enough.
+            # Check if 'other' pin was set not long ago and still is set and this pin has been high for long enough.

```

Comment:
[discussion] I think this edit changes the meaning of "set". It was intended to mean if a gpio pin value was provided for `_other` (i.e. `_handler_both()` has been called), but now it seems to change to mean if the button associated with that `_other` was recently pressed. With that understanding I'm okay with expanding the docs to refer to the pressed value of `_other` if it is "set", or using different terminology to make it more clear that `_other` has been provided, but I want to make sure we're not loosing the original documentation.